addons/languages/german/pack/src/main/res/xml/de_adnw.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016, kertase, 2020, chrbauer
```

Comment:
@chrbauer I need to change the license to Apache2. See: https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/CONTRIBUTING.md#license

I would prefer if you submit a new PR that removes the license header.

Thank you!

Reply:
https://github.com/AnySoftKeyboard/AnySoftKeyboard/pull/2674